code/src/java/pcgen/gui2/converter/ConvertPanel.java

```diff
@@ -73,36 +73,7 @@ public ConvertPanel(List<ConvertSubPanel> bq)
 	{
 		super(new BorderLayout());
 		statusLabel = new JLabel();
-		TaskStrategyListener tsl = new TaskStrategyListener()
-		{
-			private String status;
-
-			private long time;
-
-			@Override
-			public void processMessage(Object owner, String string)
-			{
-				JOptionPane.showMessageDialog(null, string);
-			}
-
-			@Override
-			public void processStatus(Object source, String string)
-			{
-				status = string;
-				statusLabel.setText(string);
-			}
-
-			@Override
-			public void processActiveItem(Object source, String string)
-			{
-				long currentTime = System.currentTimeMillis();
-				if ((currentTime - time) > 100)
-				{
-					statusLabel.setText(status + " [" + string + "]");
-					time = currentTime;
-				}
-			}
-		};
+		TaskStrategyListener tsl = (source, string) -> statusLabel.setText(string);
```

Comment:
This is simply not a valid reduction - not a Functional Interface - it has 3 methods...

Reply:
I think this PR depends on other changes I have: the other two methods are unused.